packages/block-editor/src/hooks/color.js

```diff
@@ -292,7 +292,7 @@ const getLinkColorFromAttributeValue = ( colors, value ) => {
 export function ColorEdit( props ) {
 	const { name: blockName, attributes } = props;
 	// Some color settings have a special handling for deprecated flags in `useSetting`,
-	// so we can't unwrap them by doing const { ... } = useSetting('color')
+	// so we can't unwrap them by doing conwst { ... } = useSetting('color')
```

Comment:
In fact, we can't destructure any return values with `conwst { ... }`, which is an oversight on the part of the javascript working group I think 😄

Reply:
I'll propose it at the next mweeting. 😄